apps/client/src/components/qna-posting/QnaBox/index.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+import { Button, Flex, Stack, Text } from '@sickgyun/ui';
+import { useState } from 'react';
+import { QNA_CATEGORY } from '@/constants/qna-category';
+
+const QnaBox = () => {
+  const [activeCategoryIndex, setActiveCategoryIndex] = useState(0);
+
+  return (
+    <StyledQnaBox>
+      <Stack
+        direction="horizontal"
+        align="center"
+        justify="center"
+        style={{ height: '90px', borderBottom: '1px solid white' }}
```

Comment:
```suggestion
        style={{ height: '90px', borderBottom: `1px solid ${colors.white}` }}
```

혹시 모를 예외 상황을 대비하여 colors를 사용해주세요

Reply:
넵